backend/src/main/java/hanglog/trip/domain/type/ItemType.java

```diff
@@ -1,6 +1,21 @@
 package hanglog.trip.domain.type;
 
+import java.util.Arrays;
+
 public enum ItemType {
-    SPOT,
-    NON_SPOT
+
+    SPOT(true),
+    NON_SPOT(false);
+
+    private final Boolean isSpot;
+
+    ItemType(final boolean isSpot) {
+        this.isSpot = isSpot;
+    }
+
+    public static ItemType getItemTypeByIsSpot(boolean isSpot) {
+        return Arrays.stream(ItemType.values()).filter(
+                itemType -> itemType.isSpot.equals(isSpot)
+        ).findFirst().get();
+    }
```

Comment:
이 부분 `get()` 메소드에서 `optional` 관련 워닝 뜨는데, 없으면 에러 처리해주는 로직도 같이 넣어주면 좋을 것 같아요!
그리고 이 부분 수정하면서 개행 변경과 메소드 명 `get`으로 가지가도 좋을 것 같아요
홍고가 선택해서 반영해주세요 ~! 

```suggestion
    public static ItemType get(boolean isSpot) {
        return Arrays.stream(ItemType.values())
                .filter(type -> type.isSpot.equals(isSpot))
                .findFirst()
                .orElseThrow(IllegalArgumentException::new);
    }
```

Reply:
if문으로 반영완!